nevergrad/functions/photonics/core.py

```diff
@@ -53,6 +53,12 @@ def _make_parametrization(
     if name == "bragg":
         shape = (2, dimension // 2)
         bounds = [(2, 3), (30, 180)]
+    elif name == "cf_photosic_realistic":
+        shape = (2, dimension // 2)
+        bounds = [(1, 9), (30, 180)]
+    elif name == "cf_photosic_reference":
+        shape = (1, dimension)
+        bounds = [(30, 180)]
```

Comment:
@Ellawin @AnMoreau what should be the ranges of values for inputs of both functions?

Reply:
also, are they the name you want the functions to be known as?

Reply:
Yes

Reply:
@Ellawin that answers only one of the two questions :p the other is what should be the range of values for each parts of the vectors. so far `realistic` gets 1 to 9 (that you stated) for permittivity (if I recall correctly). and 30 to 180 for the other part (depth?). For `reference`, it's 30 to 180 as well for the whole vector.

Reply:
Oh yes sorry, the bounds are perfectly correct !